### ISSUE TITLE
use width tag to render cycleway

### ISSUE THREAD

**Issue by Florimondable** [cyclosm/cyclosm-cartocss-style#319]
This could be a good way to distinct primary, secondary, tertiary cycleways.

**Comment by u4y0u** [comment]
Yes, the rendering taking into account the tag 'width' would be much more realistic because some tracks of 3 meters wide unidirectional which are parallel to roads look like a small trail when they are of identical width.
